include/z64save.h

```diff
@@ -105,13 +135,13 @@ typedef struct SavePlayerData {
     /* 0x0008 */ char playerName[8];                    // "player_name"
     /* 0x0010 */ s16 healthCapacity;                    // "max_life"
     /* 0x0012 */ s16 health;                            // "now_life"
-    /* 0x0014 */ s8 magicLevel;                         // "magic_max"
-    /* 0x0015 */ s8 magic;                              // "magic_now"
+    /* 0x0014 */ s8 magicLevel; // 0 for no magic/new load, 1 for magic, 2 for double magic "magic_max"
```

Comment:
These original names look kind of weird now. Before, they were nicely aligned with each other. Now, the ones with comments are shifted pretty far to the side, and there's no punctuation (comma, period, semicolon, etc) to signal that the comment is over and the original name starts here. I'm not sure how best to handle this sort of thing, but it's only going to get worse as we document more of these fields. Maybe something like `...2 for double magic. Originally "magic_max"`? Though that wouldn't fix the alignment thing.

Reply:
These struct members will need to be documented eventually, just having only the original names was more of a stepping stone. see `z64actor.h` (specifically `Actor`) which is better documented, and that's presumably how these structs will end up.